js/adapt-contrib-resources.js

```diff
@@ -29,7 +29,8 @@ define([
         var drawerObject = {
             title: courseResources.title,
             description: courseResources.description,
-            className: 'resources-drawer'
+            className: 'resources-drawer',
+            drawerOrder: courseResources._drawerOrder
```

Comment:
Can this be changed to this?
`drawerOrder: courseResources._drawerOrder || 0`

Just a little bit of defensive code in case the `_drawerOrder` property doesn't exist, which it wont for older courses.